pkg/gen/input/ami/internal/file/ami.go

```diff
@@ -0,0 +1,35 @@
+package file
```

Comment:
This is a file template file. It's clean and simple. Shows the template and the parameters it uses.